src/org/omegat/core/dictionaries/LingvoDSL.java

```diff
@@ -96,11 +107,19 @@ private void readDslFile(File file) throws IOException {
             }
         }
 
+        private String replaceTag(final String line) {
+            String result = line;
+            for (RE re : RE_LIST) {
+                result = result.replaceAll(re.regex, re.replacement);
```

Comment:
is it better to use `re.pattern.matcher(result).replaceAll(re.replacement)` with pre-compiled regexes?

Reply:
done